### PR TITLE
Add VaroVision.EU

### PR DESCRIPTION
**Pull request by Kalibrier**
## Type of change

- [ ] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [ ] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [ ] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [ ] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/varovision.eu/manifest.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "server_name": "VaroVision.EU",
```
Comment (DoJapHD) [inline minecraft_servers/varovision.eu/manifest.json:2]: Just use lowercase `varovision` and change the folder name to that, too

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Comment by DoJapHD** [comment]
Server is not reachable

**Comment by Kalibrier** [comment]
Please add the server

File: minecraft_servers/varovision.eu/manifest.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "server_name": "varovision.eu",
```
Comment (DoJapHD) [inline minecraft_servers/varovision.eu/manifest.json:2]: As already mentioned before, only use "varovision"!

**Review by DoJapHD — CHANGES_REQUESTED** [review]
Server has to be publicly available

**Comment by Kalibrier** [comment]
I think I did it correctly now

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**